apps/src/StudioApp.js

```diff
@@ -498,7 +498,8 @@ StudioApp.prototype.init = function (config) {
 };
 
 StudioApp.prototype.initProjectTemplateWorkspaceIconCallout = function () {
-  if (getStore().getState().pageConstants.showProjectTemplateWorkspaceIcon) {
+  if (getStore().getState().pageConstants.showProjectTemplateWorkspaceIcon &&
+      $("#projectTemplateWorkspaceIcon").is(":visible")) {
```

Comment:
Ideally our merge would actually remove this particular change (we went with a quick fix approach to get this in for HOC, then had Chris check his more complete fix in to SN). Really, I guess we could just "undo" this change in SN as a separate PR after merging this.

Reply:
I can own making that follow up PR later